webnode/src/redux/epics/treasure-hunt-epics.js

```diff
@@ -18,15 +18,15 @@ const performPowEpic = (action$, store) => {
     .ofType(treasureHuntActions.TREASURE_HUNT_PERFORM_POW)
     .mergeMap(action => {
       const { treasureHunt } = store.getState();
-      const { dataMapHash, treasure, chunkIdx, numberOfChunks } = treasureHunt;
+      const { dataMapHash, treasure, chunkIdx } = treasureHunt;
```

Comment:
Without numberOfChunks how will we know when to stop?

Reply:
this was unused, caught by the linter.